### DISCUSSION TITLE
Force verify without email dispatch

### DISCUSSION THREAD

**Discussion by realJogicodes** [pocketbase/pocketbase#4702] [Q&A] [answered]
Hi

can I use this to create verified users? I tried to set that on the sign-up having `{ verified: true }` in my `userData` on `pb.collections("users").create(userData)` but that (thankfully) returned a 400 JSON, so that's not how to do that.

I do not want friction during signup and want to auto-verify incoming users, if their email is mistyped, so be it.

All I can think of is calling an api endpoint on my server and doing it serverside with admin credetials. That doesn't sound too smart, can this be done differently, maybe using a create user hook that's watching collections and updates it when a new one is created?

**Comment by tthill** [comment]
hmm, { verified: true } does work for me.

**Comment by ganigeorgiev** [comment, marked as answer]
The `verified` field is just a bool flag. You can ignore it if you are not going to make use of it.

There are not enough details to understand what you are trying to do and why so I'm not sure that I can recommend it, but here is one example to manually set the `verified` with the JS `pb_hooks`:

```js
onModelBeforeCreate((e) => {
    e.model.set("verified", true)
}, "users")
```

If you need further help, please try to elaborate a little more on your use case.

**Reply by realJogicodes** [reply]
This did it for me, thanks.
